src/services/azureFunctionsService.ts

```diff
@@ -68,7 +68,25 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 		}
 		let projectFile = await azureFunctionUtils.getAzureFunctionProject();
 		if (!projectFile) {
-			vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened);
+			let projectCreate = await vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened, constants.learnMore, constants.createProject);
+			if (projectCreate === constants.learnMore) {
+				vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(constants.sqlBindingsDoc));
+			} else if (projectCreate === constants.createProject) {
+				// ask what folder the project should be created in
+				const options: vscode.OpenDialogOptions = {
+					title: constants.selectFolder,
+					canSelectMany: false,
+					openLabel: 'Select',
+					canSelectFiles: false,
+					canSelectFolders: true
+				};
+				// create the project based on the folder selected
+				await vscode.window.showOpenDialog(options).then(fileUri => {
+					if (fileUri && fileUri[0]) {
+						azureFunctionApi.createFunction({ folderPath: fileUri[0].fsPath });
+					}
+				});
+			}
 			return;
```

Comment:
If they create a new project then we should be able to go loop back and call getAzureFunctionProject again to save them having to re-run the command.

Reply:
not sure if this is feasible for most cases since there are instances where the create function needs to reload the window and therefore we lose the context of the connection string and such.

Reply:
Yeah, there's very little we can do there. But no reason not to try - since the window doesn't need to reload unless this is the first folder being added to the workspace. 

Anyways I'll file an improvement for it. This isn't something we need off the bat.

Reply:
Yup agreed can look into this further as well.